environment.yaml

```diff
@@ -1,116 +1,149 @@
+- notebook=5.4.0=py27_0
+- numpy=1.14.0=py27h3dfced4_1
+- openssl=1.0.2n=hb7f436b_0
+- pandas=0.22.0=py27hf484d3e_0
+- pandocfilters=1.4.2=py27h428e1e5_1
+- pango=1.41.0=hd475d92_0
+- pathlib2=2.3.0=py27h6e9d198_0
+- patsy=0.5.0=py27_0
+- pcre=8.41=hc27e229_1
+- pexpect=4.3.1=py27_0
+- pickleshare=0.7.4=py27h09770e1_0
+- pip=9.0.1=py27ha730c48_4
+- pixman=0.34.0=hceecf20_3
+- prompt_toolkit=1.0.15=py27h1b593e1_0
+- ptyprocess=0.5.2=py27h4ccb14c_0
+- pycparser=2.18=py27hefa08c5_1
+- pydot=1.2.4=py27_0
+- pygments=2.2.0=py27h4a8b6f5_0
+- pyopenssl=17.5.0=py27hcee3be0_0
+- pyparsing=2.2.0=py27hf1513f8_1
+- pyqt=5.6.0=py27h4b1e83c_5
+- pysocks=1.6.7=py27he2db6d2_1
+- python=2.7.14=h1571d57_29
+- pytz=2017.3=py27h001bace_0
+- pyyaml=3.12=py27h2d70dd7_1
+- pyzmq=16.0.3=py27hc579512_0
+- qt=5.6.2=h974d657_12
+- readline=7.0=ha6073c6_4
+- requests=2.18.4=py27hc5b0589_1
+- scandir=1.6=py27hf7388dc_0
+- scipy=1.0.0=py27hf5f0f52_0
+- seaborn=0.8.1=py27h633ea1e_0
+- send2trash=1.4.2=py27_0
+- setuptools=38.4.0=py27_0
+- simplegeneric=0.8.1=py27_2
+- sip=4.18.1=py27he9ba0ab_2
+- six=1.11.0=py27h5f960f1_1
+- sqlite=3.22.0=h1bed415_0
+- ssl_match_hostname=3.5.0.1=py27h4ec10b9_2
+- statsmodels=0.8.0=py27hc87d62d_0
+- subprocess32=3.2.7=py27h373dbce_0
+- terminado=0.8.1=py27_1
+- testpath=0.3.1=py27hc38d2c4_0
+- tk=8.6.7=hc745277_3
+- tornado=4.5.3=py27_0
+- traitlets=4.3.2=py27hd6ce930_0
+- urllib3=1.22=py27ha55213b_0
+- wcwidth=0.1.7=py27h9e3e1ab_0
+- webencodings=0.5.1=py27hff10b21_1
+- wheel=0.30.0=py27h2bc6bb2_1
+- xz=5.2.3=h55aa19d_2
+- yaml=0.1.7=had09818_2
+- zeromq=4.2.2=hbedb6e5_2
+- zlib=1.2.11=ha838bed_2
+- gmp=5.1.2=2
+- libtool=2.4.2=0
+- pandoc=1.12.3=2
+- python-dateutil=2.2=py27_0
+- singledispatch=3.4.0.3=py27_1
 - pip:
-  - async==0.6.1
+  - backports-abc==0.5
+  - backports.functools-lru-cache==1.4
   - backports.shutil-get-terminal-size==1.0.0
-  - backports.ssl-match-hostname==3.4.0.2
-  - beautifulsoup==3.2.1
-  - beautifulsoup4==4.3.2
-  - bigbang (/home/sb/projects/bigbang)==0.1
-  - chardet==2.3.0
-  - common==0.1.1
-  - community==1.0.0b1
-  - flake8==2.2.5
-  - flask==0.10.1
-  - gitdb==0.5.4
-  - gitpython==0.3.2rc1
-  - html2text==2015.2.18
-  - ipython-genutils==0.1.0
-  - itsdangerous==0.24
-  - jupyter-client==4.2.2
-  - jupyter-core==4.1.0
-  - louvain==0.5.2
-  - mccabe==0.2.1
-  - mpmath==0.19
-  - pep8==1.5.7
-  - powerlaw==1.3.4
-  - pyflakes==0.8.1
-  - python-igraph==0.7.1.post4
-  - python-levenshtein==0.11.2
-  - seaborn==0.5.1
-  - smmap==0.8.2
+  - backports.ssl-match-hostname==3.5.0.1
+  - bigbang (/home/sb/projects/bigbang-multi/bigbang)==0.1
+  - common==0.1.2
+  - gender-detector==0.1.0
+  - gitdb2==2.0.3
+  - gitpython==2.1.8
+  - html2text==2018.1.9
+  - ipython-genutils==0.2.0
+  - jupyter-client==5.2.2
+  - jupyter-core==4.4.0
+  - powerlaw==1.4.3
+  - prompt-toolkit==1.0.15
+  - python-levenshtein==0.12.0
+  - smmap2==2.0.3
   - snakefood==1.4
-  - werkzeug==0.10.4
+prefix: /home/sb/anaconda3/envs/bigbang
```

Comment:
This doesn't seem generally applicable.

Reply:
I guess this is a known bug, and supposedly it doesn't actually have any effect when others use it, but we should also be able to safely remove it.

https://stackoverflow.com/questions/41274007/anaconda-export-environment-file

Reply:
Thanks for catching this.

I think the right thing to do is:

- remove that prefix so the .yml file is generic
- document the process for creating the environment file without the prefix